src/index.js

```diff
@@ -216,7 +230,13 @@ function generatePrimaryColors() {
 			700: "var(--mantine-primary-color-7)",
 			800: "var(--mantine-primary-color-8)",
 			900: "var(--mantine-primary-color-9)",
-			DEFAULT: "var(--mantine-primary-color-6)",
+			light: mantinePrimaryShade && mantinePrimaryShade.light
+				? `var(--mantine-primary-color-${mantinePrimaryShade.light}-light)`
+				: undefined,
+			dark: mantinePrimaryShade && mantinePrimaryShade.dark
+				? `var(--mantine-primary-color-${mantinePrimaryShade.dark}-dark)`
+				: undefined,
+			DEFAULT: `var(--mantine-primary-color-${parsedMantinePrimaryShade})`,
```

Comment:
```
DEFAULT: var(--mantine-primary-color-filled)
```

Reply:
Indeed much simpler and  more suited solution!